src/kOS.Safe/Execution/CPU.cs

```diff
@@ -704,16 +695,17 @@ public object GetValue(object testValue, bool barewordOkay = false)
             // If it's a variable, meaning it starts with "$" but
             // does NOT have a value like $<.....>, which are special
             // flags used internally:
```

Comment:
Given that you flipped the meaning of the 'if' condition below to mean the opposite, that makes this comment misleading now because it's describing the old if condition and you didn't edit it to match.

But this all may be moot as I expect @hvacengi gutted this whole check in favor of something that doesn't need to examine the string prefixes anyway.